s3torchconnectorclient/python/src/s3torchconnectorclient/_mountpoint_s3_client.pyi

```diff
@@ -35,6 +35,7 @@ class MountpointS3Client:
     ) -> ListObjectStream: ...
     def head_object(self, bucket: str, key: str) -> ObjectInfo: ...
     def delete_object(self, bucket: str, key: str) -> None: ...
+    def copy_object(self, src_bucket: str, src_key: str, dst_bucket: str, dst_prefix: str) -> ObjectInfo: ...
```

Comment:
The rust implementation doesn't return an `ObjectInfo`

Reply:
Indeed, will return `None` instead, oversight